Apps/Inc/main.c

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "BSP_CAN.h"
+#include "config.h"
+// #include "ALEX FOLDER " ==> will hold checkVoltage() function
+
+int main(void){
+while(checkVolatge() > 0){
+ BSP_CAN_Write(0x10B, checkVoltage() )     //send message to CAN Controller 
+    }
```

Comment:
```suggestion
    while(checkVolatge() > 0){
        BSP_CAN_Write(0x10B, checkVoltage() )     //send message to CAN Controller 
    }
```
Generally, we indent code blocks with 4 spaces like this. (This is style, not syntax)

Reply:
Got it!